cmd/brew-portable-package.rb

```diff
@@ -1,28 +0,0 @@
-# Build a portable Homebrew utility bottle.
-#
-# Usage: brew portable-package <formula> [formula...]
-
-odie "Need to specify at least one portable formula!" if ARGV.empty?
-
-if RUBY_VERSION.split(".").first.to_i < 2
-  safe_system "brew", "install", "ruby"
-  ENV["HOMEBREW_RUBY_PATH"] = "#{HOMEBREW_PREFIX}/bin/ruby"
-end
-
-ENV["HOMEBREW_PREFER_64_BIT"] = "1"
```

Comment:
This is already documented by README.md as the beginning part of invoking command.

I also don't think this should be added in `test-bot`. As in fact, you can build 32bit only portable files if it's requested.